.changeset/rotten-eagles-hope.md

```diff
@@ -0,0 +1,7 @@
+---
+"@frameless/strapi-admin-extensions": major
+"@frameless/overige-objecten-api": major
+"@frameless/pdc-dashboard": major
+---
+
+Lijn de VAC-trefwoorden uit met de product- en internveld collectie ([GitHub Issue frameless/strapi#1067](https://github.com/frameless/strapi/issues/1067)).
```

Comment:
Zoals het er nu staat moet je weten hoe het was, je kunt ook gelijk duidelijk maken wat de verandering is.

```suggestion
De trefwoorden van de VAC is niet meer een lijst van trefwoorden, maar een tekstinvoerveld voor komma-gescheiden trefwoorden. De trefwoorden van VAC werken nu hetzelfde als voor de collecties voor Producten en Interne Velden ([GitHub Issue frameless/strapi#1067](https://github.com/frameless/strapi/issues/1067)).
```